doc/api/stream.md

```diff
@@ -621,7 +621,7 @@ added:
 Number of times [`writable.uncork()`][stream-uncork] needs to be
 called in order to fully uncork the stream.
 
-##### `writable.writableErrored`
+##### `writable.errored`
```

Comment:
needs changes: entries?

Reply:
These props have never made it to any release... is that really necessary?